db/schema/transaction.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const user = require('user.js')
+
+const transactionSchema = new mongoose.Schema({
+    uuid: String,
+    date: String,
+    items: Array,

```

Comment:
Is there a way to specify the type attribute for items as an array of item.js? (as opposed to an array of Strings)

E.g. `items: [item]`